client/js/components/map/publicdetail/Map.vue

```diff
@@ -420,7 +420,12 @@ export default {
         const firstTogglableLayer = this.baseLayers.find(layer => {
           return layer.getProperties().isEnabled === true && layer.getProperties().title !== 'customBaselayerDanmark'
         })
-        firstTogglableLayer.setVisible(true)
+
+        if(typeof firstTogglableLayer !== 'undefined') {
+          firstTogglableLayer.setVisible(true)
+        } else {
+          console.warn('There is no published baseLayer')
```

Comment:
```suggestion
          console.warn('There is no published baseLayer, please go to the admin area and set a baselayer as "published"')
```

Reply:
Thank you for the advice. done https://github.com/demos-europe/demosplan-core/pull/626/commits/4c4664def66686d2a8e8745ed430a4ecf1554fc5